ldap/servers/slapd/log.c

```diff
@@ -3337,12 +3338,11 @@ log__open_accesslogfile(int logfile_state, int locked)
         return LOG_UNABLE_TO_OPENFILE;
     }
 
-
     /* write the header in the log */
     now = slapi_current_utc_time();
     log_convert_time(now, tbuf, 2 /* long */);
     PR_snprintf(buffer, sizeof(buffer), "LOGINFO:Log file created at: %s (%lu)\n", tbuf, now);
-    LOG_WRITE(fpinfo, buffer, strlen(buffer), 0);
+    log_write(fpinfo, buffer, strlen(buffer), 0);
```

Comment:
check the return code ?

Reply:
@tbordaz No, errors here are acceptable.  Aborting on an error breaks things - trust me I tried it before and the CI test fail.  We really only need the error checking for the error log where we need to shutdown if we can't log to it.  In my effort to reduce duplicate code we have to return an "rc" but it doesn't really apply to most logs.

 In the cases you mentioned it is the rotationinfo file that it is working on and errors are expected under certain conditions.  Again, I'm trying not to change the existing behavior because it break things.